### PR TITLE
#162414226 Setup react router

### PR DESCRIPTION
**Pull request by Makwe-O**
#### What does this PR do?
Setup React Router Version 4.

#### Description of Task to be completed?
* Install react router version 4
* Edit webpack config file to have historyApiFallback set to true
* Create Landing and 404 page

#### What are the relevant pivotal tracker stories?
[Setup React Router](https://www.pivotaltracker.com/story/show/162414226)

### CODE REVIEW

File: .gitIgnore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
+#Ignore dist folder
+dist
+
+.DS_Store
```
Comment (Bobsar0) [inline .gitIgnore]: Empty line needed here

File: src/Reducer/rootReducer.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+import placeholderReducer from './placeholder';
+
+const reducers = combineReducers({
+  placeholderReducer,
+});
+
+ export default reducers;
+
```
Comment (ebzeal) [inline src/Reducer/rootReducer.js:9]: add empty line

**Review by ebzeal — COMMENTED** [review]
Nice job! Review the conversations on your file changes

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    contentBase: path.join(__dirname, 'src'),
+    port: 4000,
+    historyApiFallback: true
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, 'src', 'index.html')
+    })
+  ],
+  resolve: { extensions: ['.js', '.jsx'] }
```
Comment (jesseinit) [inline webpack.config.js:60]: Good job. Maks

**Comment by jesseinit** [comment]
Good Job Maks. Looks good to me.

File: webpack.config.js (hunk 3 above)
Comment (Makwe-O) [reply webpack.config.js:60]: Thanks

**Pull request merged**